GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -569,7 +569,6 @@ bool Renderer::isShader3d(const Material* curr_material) {
     case Material::ShaderType::TEXTURE_SHADER:
     case Material::ShaderType::EXTERNAL_RENDERER_SHADER:
     case Material::ShaderType::ASSIMP_SHADER:
```

Comment:
looks like you need to rebase.  seems you're missing the lightmap shader that was added recently.